### ISSUE TITLE
Add the possibility to link to a homeserver

### ISSUE THREAD

**Issue by ludwigbald** [matrix-org/matrix.to#80]
For example: https://matrix.to/#/matrix.org

What would that mean?
- Start a client by clicking on that link, and it prompts registration/login on that homeserver.

Use case: Jenny wants her friends to join her matrix server, but she wants them to chose their own client. Rather than explaining they need to set their client to use her server, which might be a different procedure for every client, she could simply send one link to all of them.

**Comment by danger89** [comment]
I fully agree, that would be the whole point of federation right..

**Comment by john-cena-141** [comment]
Yes, we need this.

**Comment by danger89** [comment]
@JorikSchellekens  Is this on the roadmap anywhere planned?

**Comment by pReya** [comment]
Yes, this is absolutely needed. Is there a way / defined link schema to start a client signup process with a specific homeserver? This would need to be supported by the clients, too, not just matrix.to.

**Comment by MadLittleMods** [comment]
Related to https://github.com/matrix-org/matrix.to/issues/15
